docs/wiki/get-started/More-with-CSP.md

```diff
@@ -0,0 +1,141 @@
+csp offers several built-in generic, mathematical, and statistical nodes that are often required in streaming workflows. This allows you to write applications quickly, and update and expand them as required by including new nodes.
+
+In this tutorial, you will calculate the [volume weighted average price (VWAP)](https://www.investopedia.com/terms/v/vwap.asp) and the profit and loss (PnL) of the trade, on a stream of trades. Check out the complete example: [trade profit-and-loss example](examples/01_basics/e4_trade_pnl.py)
+
+## Compound inputs with `csp.Struct`
+
+A Trade consists of multiple parts: price of a share, quantity of shares, and indication of a buying or a selling transaction.
+
+`struct` is a useful data type for this type of information. In csp, you can use [`csp.Struct`](csp.Struct-API), a higher-level data type that can be defined with type-annotated values.
+
+```python
+class Trade(csp.Struct):
```

Comment:
We really should be able to come up with a non-finance example 😅 
Maybe we can use my machine monitoring example as a starting point: https://github.com/timkpaine/timkpaine/blob/main/talks/perspective2024/csp_nodes.py

Reply:
We discussed this on Wednesday, and decided to update the profit and loss example to be more commerce/retail oreiented instead. We can use cost & selling price of products to calc pnl, and demostrate similar set of baselib nodes. Does that sound OK to you? 

I can make this upadte in a follow-up PR because we want to merge this PR before the blog post is published early next week.